redfish-core/lib/log_services.hpp

```diff
@@ -4486,6 +4517,11 @@ inline void getRedfishUriByDbusObjPath(
                         return;
                     }
 
+                    // tuple: assembly parent service name, object path, and
+                    // interface
+                    std::tuple<std::string, sdbusplus::message::object_path,
```

Comment:
Don't you need #include <tuple> ?

Reply:
Ack, I thought we no need since bmcweb is a kind of header-specific implementation, not a CPP file because added and removed no effects but, I will add to avoid dependence on the other bmcweb header file.